### PR TITLE
Small refactoring, more to come

### PR DESCRIPTION
**Pull request by cezaris13**
Updated gradle version to the newest, fixed several warnings and errors

### CODE REVIEW

File: app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.java
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.java:1]: inconsistent use of coding standard.
in some rows for example 29 the equal sign is separated by one space on the right, in others like row 38 it is separated on both sides.

File: app/src/main/java/com/example/pijus/chemijosprojektas/balinamoji1.java
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/balinamoji1.java:1]: rows 25-29 have 2 spaces after the equal sign.

File: app/src/main/java/com/example/pijus/chemijosprojektas/dezodorantas2.java [hunk 1]
```diff
@@ -23,41 +22,38 @@ public class dezodorantas2 extends AppCompatActivity {
...
-        treciasvonia=(EditText)findViewById(R.id.ivestisd3);
-        ketvirtasvonia=(EditText)findViewById(R.id.ivestisd4);
-        atsakymas=(TextView)findViewById(R.id.textViewd2);
-        voniaskaiciuoti.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                atsakymas.setText("");
-                if(pirmasvonia.getText().toString().isEmpty()||antrasvonia.getText().toString().isEmpty()||treciasvonia.getText().toString().isEmpty()|ketvirtasvonia.getText().toString().isEmpty()){
-                    Toast.makeText(getApplicationContext(), "Prašome užpildyti visus langelius",1000).show();
+        voniaskaiciuoti= findViewById(R.id.buttond3);
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/dezodorantas2.java]: no spaces before equal signs

File: app/src/main/java/com/example/pijus/chemijosprojektas/losjonas2.java [hunk 2]
```diff
@@ -25,46 +24,42 @@ public class losjonas2 extends AppCompatActivity {
...
-                        min = vanduo3/lasas;
-                    }
-                    if (min > vanduo4/alavijo) {
-                        min = vanduo4/alavijo;
-                    }
-                    if (min > vanduo5/E) {
-                        min = vanduo5/E;
-                    }
-                    atsakymas.setText(String.valueOf(String.format("%.2f",min)) + "porcijų");
+        voniaskaiciuoti= findViewById(R.id.buttonl3);
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/losjonas2.java]: no spaces before equal signs

File: app/src/main/java/com/example/pijus/chemijosprojektas/pasta1.java [hunk 3]
```diff
@@ -24,37 +22,26 @@ public class pasta1 extends AppCompatActivity {
...
-                int data=Integer.decode(ivestispp.getText().toString());
-                rezultataspp.setText(String.valueOf(data*milteliai)+" g kepimo miltelių");
-                rezultataspp2.setText(String.valueOf(data*druska)+" g druskos");
-                rezultataspp3.setText(String.valueOf(data*glizerolis)+" ml glicerolio");
-                receptaspp.setText(" Darbo eiga: \n" +
-                        "1. Sumašykite miltelius ir druską. \n" +
-                        "2. Spilkite glicerolį. \n" +
-                        "3. Pilkite iš lėto vandenį, kol gausite tinkamos konsistencijos masę.");
-                 }
+        skaiciuotipp= findViewById(R.id.skaiciuotip);
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/pasta1.java]: no spaces before equal sign

File: app/src/main/java/com/example/pijus/chemijosprojektas/pasta2.java [hunk 4]
```diff
@@ -21,36 +20,32 @@ public class pasta2 extends AppCompatActivity {
...
-                        min = vanduo1;
-                    }
-                    if (min > vanduo2/druskaa) {
-                        min = vanduo2;
-                    }
-                    if (min > vanduo3/gliceroliss) {
-                        min = vanduo3;
-                    }
-                    atsakymas.setText(String.valueOf(String.format("%.2f",min)) + "porcijų");
+        voniaskaiciuoti= findViewById(R.id.buttonp3);
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/pasta2.java]: no spaces before equal signs

File: app/src/main/java/com/example/pijus/chemijosprojektas/unknown1.java [hunk 5]
```diff
@@ -24,38 +22,25 @@ public class unknown1 extends AppCompatActivity {
...
-                rezultatasuu.setText(String.valueOf(data*miltai)+" g miltų");
-                rezultatasuu2.setText(String.valueOf(data*druska)+" g druskos");
-                rezultatasuu3.setText(String.valueOf(data*vanduooo)+" ml vandens");
-                receptasunknown.setText("Darbo eiga: \n" +
-                        "1. Iš sudėtinių dalių suminkykite tešlą. \n" +
-                        "2. Gaminkite norimą skulptūrą.");
-
-                 }
-            }
+        skaiciuotiuu= findViewById(R.id.skaiciuotiu);
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/unknown1.java]: yet again no spaces before equal signs

File: app/src/main/java/com/example/pijus/chemijosprojektas/unknown2.java [hunk 6]
```diff
@@ -21,36 +20,33 @@ public class unknown2 extends AppCompatActivity {
...
-                        min = vanduo1/mltai;
-                    }
-                    if (min > vanduo2/vanduoooo) {
-                        min = vanduo2/vanduoooo;
-                    }
-                    if (min > vanduo3/druska) {
-                        min = vanduo3/druska;
-                    }
-                    atsakymas.setText(String.valueOf(String.format("%.2f", min)) + "porcijų");
+        voniaskaiciuoti= findViewById(R.id.buttonu3);
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/unknown2.java]: no spaces before equal signs

File: app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.kt [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+import com.example.pijus.chemijosprojektas.Quantity.balzamas1
+import com.example.pijus.chemijosprojektas.Quantity.dezodorantas1
+import com.example.pijus.chemijosprojektas.Quantity.kauke1
+import com.example.pijus.chemijosprojektas.Quantity.losjonas1
+import com.example.pijus.chemijosprojektas.Quantity.pasta1
+import com.example.pijus.chemijosprojektas.Quantity.unknown1
+import com.example.pijus.chemijosprojektas.Quantity.vonioszele1
+
+open class MainActivity : AppCompatActivity() {
+    private val toastMessage = "prasome pasirinkti nors viena recepta"
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.kt]: toast message here is written without using lithuanian characters whereas later they are used. fix it.

File: app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/losjonas11.kt [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+            } else {
+                val data = Integer.decode(ivestisll?.text.toString())
+                rezultatasll?.text = (data * aliejus).toString() + " g aliejaus"
+                rezultatasll2?.text = (data * vaskas).toString() + " g bičių vaško"
+                rezultatasll3?.text = (data * lasas).toString() + " lašų eterinio aliejaus"
+                rezultatasll4?.text = (data * alavijo).toString() + " lašų alavijo"
+                rezultatasll5?.text = (data * E).toString() + " tablečių vitamino E"
+                receptaslos?.text = "Darbo eiga: \n" +
+                        "1. Aliejuje ištirpinti vašką. Neleisti užvirti.\n" +
+                        "2.  Į šia masę pilti vitaminą E, eterinį aliejų ir alaviją. \n" +
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/losjonas11.kt]: extra space after number.

File: app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/pasta11.kt [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+                Toast.makeText(applicationContext, "Prašome įvesti kiekį", Toast.LENGTH_SHORT)
+                    .show()
+            } else {
+                val data = Integer.decode(ivestispp?.text.toString())
+                rezultataspp?.text = (data * milteliai).toString() + " g kepimo miltelių"
+                rezultataspp2?.text = (data * druska).toString() + " g druskos"
+                rezultataspp3?.text = (data * glizerolis).toString() + " ml glicerolio"
+                receptaspp?.text = " Darbo eiga: \n" +
+                        "1. Sumašykite miltelius ir druską. \n" +
+                        "2. Spilkite glicerolį. \n" +
```
Comment (pagn4) [inline app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/pasta11.kt]: "spilkite"

File: app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.java
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.java:1]: reformatted code.

File: app/src/main/java/com/example/pijus/chemijosprojektas/balinamoji1.java
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/balinamoji1.java:1]: reformated code.

File: app/src/main/java/com/example/pijus/chemijosprojektas/dezodorantas2.java (hunk 1 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/dezodorantas2.java]: Fixed now.

File: app/src/main/java/com/example/pijus/chemijosprojektas/losjonas2.java (hunk 2 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/losjonas2.java]: Fxed now.

File: app/src/main/java/com/example/pijus/chemijosprojektas/pasta1.java (hunk 3 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/pasta1.java]: Fixed now.

File: app/src/main/java/com/example/pijus/chemijosprojektas/pasta2.java (hunk 4 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/pasta2.java]: Fixed now.

File: app/src/main/java/com/example/pijus/chemijosprojektas/unknown2.java (hunk 6 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/unknown2.java]: Fixed now.

File: app/src/main/java/com/example/pijus/chemijosprojektas/unknown1.java (hunk 5 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/unknown1.java]: Fixed now.

File: app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/pasta11.kt (hunk 9 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/pasta11.kt]: fixed.

File: app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/losjonas11.kt (hunk 8 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/Quantity/losjonas11.kt]: Fixed.

File: app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.kt (hunk 7 above)
Comment (cezaris13) [reply app/src/main/java/com/example/pijus/chemijosprojektas/MainActivity.kt]: Fixed.

**Review by pagn4 — APPROVED** [review]
Didn't find any problems. Well done!

**Pull request merged**